src/components/DPConnectorDetails.js

```diff
@@ -21,17 +30,41 @@ class DPConnectorDetails extends Component {
 		console.log('connectorJson ==>', connectorJson);
 		console.log('swimlanesData ==>', swimlanesData);
 		console.log('connectorsData ==>', connectorsData);
+		const connector = swimlanesData.connector_details[connectorJson.dispName];
+		console.log('connector ==>', connector);		
+		const connectorData = _.find(connectorsData, {id: connector.type});
+		console.log('connectorData ==>', connectorData);
+		const derivedDetails = Object.keys(connector.derivedDetails).map(key => (
+			<div><b>{key}:</b> {connector.derivedDetails[key]}<br/></div>))
+
+		const fullDetails = Object.keys(connector.fullConfig).map(key => (
+			<div><b>{key}:</b> {connector.fullConfig[key]}<br/></div>))
 		return (
 			<Modal className="animated zoomIn" isOpen={modal} toggle={toggle}>
-				<ModalHeader toggle={toggle}>{connectorJson.dispName}</ModalHeader>
+				<ModalHeader toggle={toggle}>{connector.dispName}</ModalHeader>
 				{
 					(connectorsLoading) ? 'loading...'
 						: (<ModalBody>
-								Type: {connectorJson.type}<br/>
-								Database: {connectorJson.database}<br/>
-								Status: {connectorJson.status}<br/>
+								<b>Name:</b> {_.get(connectorData, 'disp_name')}<br/>
+								<b>Version:</b> {_.get(connectorData, 'version')}<br/>
+								<b>Description:</b> {_.get(connectorData, 'description')}<br/>
+								{connector.database ? <div><b>Database:</b> {connector.database}<br/></div> : null}
+								<b>Status:</b> {connector.connectorStatusStr}<br/><br/>
+								<h5>Configuration:</h5>
+								{derivedDetails}
+
 							</ModalBody>)
 				}
+
+				<button className="toggle-button" color="primary" onClick={this.toggleCollapse} style={{ marginBottom: '1rem' }}>More</button>
```

Comment:
You can include this inside of the `<ModalBody`

Reply:
I tried including that inside ModalBody, but then it doesn't align properly.